console/src/main/java/org/georchestra/console/events/DefaultPropsInitializer.java

```diff
@@ -0,0 +1,27 @@
+package org.georchestra.console.events;
+
+import org.apache.log4j.Logger;
+import org.springframework.context.ApplicationContextInitializer;
+import org.springframework.context.ConfigurableApplicationContext;
+import org.springframework.core.io.FileSystemResource;
+import org.springframework.core.io.support.ResourcePropertySource;
+
+import java.io.IOException;
+
+public class DefaultPropsInitializer implements ApplicationContextInitializer<ConfigurableApplicationContext> {
+
+    private static Logger LOGGER = Logger.getLogger(DefaultPropsInitializer.class);
+
+    @Override
+    public void initialize(ConfigurableApplicationContext applicationContext) {
+        try {
+            applicationContext.getEnvironment().getPropertySources()
+                    .addFirst(new ResourcePropertySource(new FileSystemResource(
+                            System.getProperty("georchestra.datadir", "etc/georchestra") + "/default.properties")));
+        } catch (IOException e) {
+            LOGGER.debug("Cannot find default props file");
+            LOGGER.error(e.getStackTrace());
```

Comment:
maybe LOGGER.error for the first one as well ?

Reply:
or even `LOGGER.error("Cannot find ...", e);`

Reply:
you can still put the exception as argument from the Logger.error call I'd expect.